blog/_posts/2016-09-30-The-end-of-an-amazing-summer(winter) of code.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The end of an amazing summer(winter) of code.
+layout: post
+created_at: Fri sept 30 2016
+permalink: blog/2016-09-30-The-end-of-an-amazing-summer(winter) of code.
```

Comment:
hi ramya, you need hyphens and no spaces between all of the words of the permalink. Also, periods can not be used in the permalink, because that messes with the URL. Could you change that please?